src/js/console.js

```diff
@@ -23,33 +31,98 @@ function formattedMessages(messages) {
     return message;
   });
   return formattedMessages;
+}; */
+function formattedMessages(message) {
+  if (typeof message === 'string') {
+    return message;
+  }
```

Comment:
change the order and shift the stack to the top since stack trace recording is important

Reply:
changed